src/reynir_correct/errtokenizer.py

```diff
@@ -1907,10 +1925,13 @@ def is_valid_suggestion(
         if token.error_code:
             # This token already has an associated error and eventual correction:
             # let it be
-            yield token
-            # We're now within a sentence
-            at_sentence_start = False
-            continue
+            if token.error_code in {"Z001", "Z002"}:
+                pass
```

Comment:
This looks iffy. Definitely correct? Is at_sentence_start set correctly to False in this case also?

Reply:
The former version of the code checked if the token contained an error. If so, it yielded the token and was no longer at the start of the sentence.
The only addition here is to keep looking for errors if the token error is a capitalization. This allows us to also correct spelling errors in cases such as "eg vil fleiri hesta.". We begin by correcting the cap error: "Eg vil fleiri hesta". Since we keep checking, we also find the spelling error and get: "Ég vil fleiri hesta."